crates/matrix-sdk/src/room/timeline/mod.rs

```diff
@@ -435,6 +436,18 @@ impl TimelineItem {
     }
 }
 
+impl From<EventTimelineItem> for TimelineItem {
+    fn from(item: EventTimelineItem) -> Self {
+        Self::Event(item)
+    }
+}
+
+impl From<VirtualTimelineItem> for TimelineItem {
+    fn from(item: VirtualTimelineItem) -> Self {
+        Self::Virtual(item)
+    }
+}
```

Comment:
Can this and its usages in existing code be a separate PR, to make this one a bit smaller?

Reply:
Sure, I'll do it when this is approved and I can rebase everything, if that's okay.

Reply:
Oh to be clear, a separate PR that's merged first.

Reply:
Oh it's my fault, I read "PR" and thought "commit".